src/core/core-varlink.c

```diff
@@ -295,6 +331,12 @@ int manager_varlink_init(Manager *m) {
         if (r < 0)
                 return log_error_errno(r, "Failed to bind to varlink socket: %m");
 
+        (void) mkdir_p("/run/systemd/oom", 0755);
+
+        r = varlink_server_listen_address(s, "/run/systemd/oom/io.systemd.Oomd", 0666);
```

Comment:
Uuuh yea... I wanted a way to set this as a slice property but also a reliable way to get all the cgroup paths of those enabled slices to oomd without having to poll all the slices on the system periodically. I'm open to alternatives since varlink just seemed so convenient for now.

Reply:
the service name is a misnomer though if it's systemd that listens here. maybe use /run/systemd/io.system.OutOfMemory as path?